data/json/requirements/toolsets.json

```diff
@@ -24,7 +24,13 @@
     "id": "sewing_standard",
     "type": "requirement",
     "//": "Crafting or repair of fabric items",
-    "tools": [ [ [ "sewing_kit", 1 ], [ "tailors_kit", 1 ], [ "needle_bone", 1 ], [ "needle_wood", 1 ] ] ]
+    "tools": [ [ [ "sewing_kit", 1 ], [ "tailors_kit", 1 ], [ "needle_bone", 1 ], [ "needle_wood", 1 ], [ "sewing_machine", 1 ] ] ]
+  },
+  {
+    "id": "knitting_standard",
```

Comment:
I just realized it is possible to replace `tools` with `qualities` in `sewing_standard`:

**before**:

```json
{
    "id": "sewing_standard",
    "type": "requirement",
    "//": "Crafting or repair of fabric items",
    "tools": [ [ [ "sewing_kit", 1 ], [ "tailors_kit", 1 ], [ "needle_bone", 1 ], [ "needle_wood", 1 ] ] ]
},
```
**after**:

```json
{
    "id": "sewing_standard",
    "type": "requirement",
    "//": "Crafting or repair of fabric items",
    "qualities": [ { "id": "SEW", "level": 1 } ],
},
```

That way we won't need to add new tools (like `sewing_machine` or `electric_knitter`) to toolsets.